devices/Lp3943/Lp3943.cs

```diff
@@ -0,0 +1,208 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Device.Gpio;
+using System.Device.I2c;
+using System.Threading;
+
+namespace Iot.Device.Lp3943
+{
+	/// <summary>
+	/// 16-Channel LED-Driver Lp3943
+	/// </summary>
+	public class Lp3943 : IDisposable
+	{
+		/// <summary>
+		/// Lp3943 I2C Address
+		/// </summary>
+		public const byte DefaultI2cAddress = 0x60;
+
+		private readonly int _pinReset;
+		private GpioController _controller;
```

Comment:
This one can be readonly as well?

Reply:
With the way the Dispose pattern is implemented currently, no.

Reply:
if you don't set it up to null in the Dispose, it's all ok. Did I miss something else?

Reply:
no, then Ill change it 👍